tests/fixtures/SingleString.pot

```diff
@@ -1,3 +1,2 @@
-
```

Comment:
2) react-gettext-parser compiling to pot should spit out a pot file containing an extracted string:

	      AssertionError: expected 'msgid "Translate me"\nmsgstr ""' to equal 'msgid "Translate me"\r\nmsgstr ""'
	      + expected - actual

	      -msgid "Translate me"
	      +msgid "Translate me"
	       msgstr ""